### ISSUE TITLE
config file

### ISSUE THREAD

**Issue by MagnetoYuWang** [bitshenwenxiao/SiamSlim#2]
Hello, I am a student and have benefited from reading your work. I recently tried to run this code and during the run I found that the iou loss becomes negative after a few epochs of training. I would be grateful if you could provide me with the relevant configuration file

here is my config.yaml:
`META_ARC: "siamban_alexnet"

BACKBONE:
    TYPE: "alexnet"
    PRETRAINED: ''
    TRAIN_LAYERS: ['layer1','layer2','layer3','layer4','layer5']
    TRAIN_EPOCH: 10
    LAYERS_LR: 0.1

ADJUST:
    ADJUST: False

BAN:
    BAN: True
    TYPE: 'DepthwiseBAN'
    KWARGS:
        in_channels: 192
        out_channels: 192

POINT:
    STRIDE: 8

TRACK:
    TYPE: 'SiamBANTracker'
    WINDOW_INFLUENCE: 0.4632532824922313  # VOT2018
    PENALTY_K: 0.08513642556896711  # VOT2018
    LR: 0.44418184746462425  # VOT2018
#    WINDOW_INFLUENCE: 0.334450048565355  # VOT2019
#    PENALTY_K: 0.0012159181005195463  # VOT2019
#    LR: 0.46386814967815493  # VOT2019
    EXEMPLAR_SIZE: 127
    INSTANCE_SIZE: 255
    BASE_SIZE: 0
    CONTEXT_AMOUNT: 0.5

TRAIN:
    EPOCH: 50
    START_EPOCH: 0 # 0 or resume checkpoint
    BATCH_SIZE: 64
    BASE_SIZE: 0
    NUM_WORKERS: 8
    OUTPUT_SIZE: 17
    BASE_LR: 0.005
    CLS_WEIGHT: 1.0
    LOC_WEIGHT: 1.0
    RESUME: '' # '' or 'snapshot/checkpoint_e.pth'
    IOU_TYPE: 1
    # LR:
    #     TYPE: 'log'
    #     KWARGS:
    #         start_lr: 0.005
    #         end_lr: 0.0005 # 0.0005 
    # LR_WARMUP:
    #     TYPE: 'step'
    #     EPOCH: 5
    #     KWARGS:
    #         start_lr: 0.001
    #         end_lr: 0.005
    #         step: 1
    LR:
        TYPE: 'multi-step'
        KWARGS:
            start_lr: 0.02
    LR_WARMUP:
        TYPE: 'step'
        EPOCH: 5
        KWARGS:
            start_lr: 0.0000001
            step: 1

DATASET:
    NAMES:
    - 'VID'
    - 'YOUTUBEBB'
    - 'COCO'
    - 'DET'
    # - 'GOT10K'
    # - 'LASOT'

    VIDEOS_PER_EPOCH: 600000

    TEMPLATE:
        SHIFT: 4
        SCALE: 0.05
        BLUR: 0.0
        FLIP: 0.0
        COLOR: 1.0

    SEARCH:
        SHIFT: 64
        SCALE: 0.18
        BLUR: 0.2
        FLIP: 0.0
        COLOR: 1.0

    NEG: 0.2
    GRAY: 0.0
`

here is my train log:
`[2023-04-26 22:28:10,937-rk0-train.py#189] epoch: 2
[2023-04-26 22:28:10,938-rk0-train.py#193] epoch 2 lr 0.0001379729661461215
[2023-04-26 22:28:10,938-rk0-train.py#193] epoch 2 lr 0.0013797296614612151
[2023-04-26 22:28:13,680-rk0-train.py#242] Epoch: [2][9/1171] lr: 0.001380
	batch_time: 0.249766 (0.238245)	data_time: 0.000168 (0.000970)
	cls_loss: 0.124561 (0.135507)	loc_loss: 0.490183 (0.860115)
	total_loss: 0.614743 (0.995623)	
[2023-04-26 22:28:13,680-rk0-log_helper.py#105] Progress: 1180 / 58550 [2%], Speed: 0.238 s/iter, ETA 0:03:47 (D:H:M)

[2023-04-26 22:28:20,463-rk0-train.py#242] Epoch: [2][29/1171] lr: 0.001380
	batch_time: 0.363974 (0.238587)	data_time: 0.000139 (0.000796)
	cls_loss: 0.146864 (0.135990)	loc_loss: -0.209637 (0.703984)
	total_loss: -0.062773 (0.839975)	
[2023-04-26 22:28:20,463-rk0-log_helper.py#105] Progress: 1200 / 58550 [2%], Speed: 0.239 s/iter, ETA 0:03:48 (D:H:M)

[2023-04-26 22:28:27,434-rk0-train.py#242] Epoch: [2][49/1171] lr: 0.001380
	batch_time: 0.183184 (0.241649)	data_time: 0.000158 (0.000785)
	cls_loss: 0.139943 (0.135658)	loc_loss: -1.066740 (0.412676)
	total_loss: -0.926797 (0.548334)	
[2023-04-26 22:28:27,435-rk0-log_helper.py#105] Progress: 1220 / 58550 [2%], Speed: 0.242 s/iter, ETA 0:03:50 (D:H:M)

[2023-04-26 22:28:33,660-rk0-train.py#242] Epoch: [2][69/1171] lr: 0.001380
	batch_time: 0.284950 (0.238426)	data_time: 0.000172 (0.000757)
	cls_loss: 0.116974 (0.134372)	loc_loss: -2.000886 (-0.042145)
	total_loss: -1.883912 (0.092228)	
[2023-04-26 22:28:33,670-rk0-log_helper.py#105] Progress: 1240 / 58550 [2%], Speed: 0.238 s/iter, ETA 0:03:47 (D:H:M)

[2023-04-26 22:28:40,670-rk0-train.py#242] Epoch: [2][89/1171] lr: 0.001380
	batch_time: 0.208018 (0.236102)	data_time: 0.000168 (0.000782)
	cls_loss: 0.136553 (0.133121)	loc_loss: -2.869178 (-0.715042)
	total_loss: -2.732625 (-0.581920)	
[2023-04-26 22:28:40,679-rk0-log_helper.py#105] Progress: 1260 / 58550 [2%], Speed: 0.236 s/iter, ETA 0:03:45 (D:H:M)`
